### PR TITLE
Issue #3273 support multistatments for JDBC driver

### LINKED ISSUE #3273: JDBC import database error
#### What happens?
JDBC import database error

#### To Reproduce
`@Test 
   public void importDuckDB() throws SQLException, IOException { 

       final String EXPORT_SQL_TEMPLATE = "EXPORT DATABASE '%s' (FORMAT PARQUET);"; 
       String path = "export"; 
       File export = new File(path); 
       if (export.exists()) { 
           export.delete(); 
       } 
       export.mkdir(); 
       System.out.println(export.getAbsolutePath()); 
       try (Connection conn = DriverManager.getConnection("jdbc:duckdb:db001"); Statement stmt = conn.createStatement();) { 
           stmt.execute("CREATE SCHEMA IF NOT EXISTS mydb"); 
           stmt.execute("CREATE TABLE IF NOT EXISTS mydb.student(id INTEGER,name VARCHAR,PRIMARY KEY(id))"); 
           stmt.execute(String.format(EXPORT_SQL_TEMPLATE, path)); 
       } 

       final String IMPORT_SQL_TEMPLATE = "IMPORT DATABASE '%s';"; 

       try (Connection conn = DriverManager.getConnection("jdbc:duckdb:db002"); Statement stmt = conn.createStatement();) { 
           stmt.execute(String.format(IMPORT_SQL_TEMPLATE, path)); 
       } 

   }

error info:
java.sql.SQLException: Cannot prepare multiple statements at once! 

     at org.duckdb.DuckDBNative.duckdb_jdbc_prepare(Native Method) 
     at org.duckdb.DuckDBPreparedStatement.prepare(DuckDBPreparedStatement.java:73) 
     at org.duckdb.DuckDBPreparedStatement.execute(DuckDBPreparedStatement.java:121)
`

#### Environment (please complete the following information):
 - OS: win10
 - DuckDB Version: 0.3.1
 - DuckDB Client: jdbc-0.3.1

### PR DESCRIPTION
**Pull request by melzareix**
Fixes #3273 and adds support for multiple statements in the JDBC driver.

### CODE REVIEW

**Comment by hannes** [comment]
Thanks, looks great to me. One minor thing, there are a lot of formatting changes that are unrelated to the actual changes. Would you mind removing those?

File: tools/jdbc/src/jni/duckdb_java.cpp [hunk 1]
```diff
@@ -265,13 +265,34 @@ JNIEXPORT jobject JNICALL Java_org_duckdb_DuckDBNative_duckdb_1jdbc_1prepare(JNI
                                                                              jbyteArray query_j) {
 	auto conn_ref = (Connection *)env->GetDirectBufferAddress(conn_ref_buf);
 	if (!conn_ref) {
-		env->ThrowNew(J_SQLException, "Invalid connection");
+		env->ThrowNew(env->FindClass("java/sql/SQLException"), "Invalid connection");
```
Comment (Jens-H) [inline tools/jdbc/src/jni/duckdb_java.cpp:268]: With commit ae0f79afbe9992dd9020291000a1c07c52212051 there `env->FindClass` calls can be avoided. Maybe you change this accordingly to something like: `env->ThrowNew(J_SQLException, "reason ...");`

**Comment by Mytherin** [comment]
@melzareix could you have a look at the comments from Jens?

**Comment by Mytherin** [comment]
@melzareix any updates on this PR?

**Comment by melzareix** [comment]
@Mytherin fixed the error that was failing the test should be good to go when the CI runs

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**